cvcreator/vitae/tech_skills.py

```diff
@@ -92,3 +88,49 @@ def make_skill_groups(
     output = [TechnicalSkill(title=title, values=output[title])
               for title in skills_data["allowed_labels"] if title in output]
     return output
+
+
+def get_skills(skill_categories: List[SkillCategory]) -> List[TechnicalSkill]:
+    """Process skills organized in skill categories into a list of `TechnicalSkill` objects
+
+    Args:
+        skill_categories:
+            User provided skills to distribute into groups.
+
+    Returns:
+        List of TechnicalSkill objects
+    """
+    output: List[TechnicalSkill] = []
+
+    # Get technical skills data
+    skills_data = get_skills_data()
+
+    # Process the skill categories in the CV
+    for skcat in skill_categories:
+        label = skcat.category
+        skills = skcat.technical_skills
```

Comment:
Small suggestion for readability: I propose renaming the intermediate variable to `skill_category`. It might be me that is slow, but I had to do a double take when I read the for-loop :sweat_smile: 

```suggestion
    for skill_category in skill_categories:
        label = skill_category.category
        skills = skill_category.technical_skills
```